src/test_l1_hdmi_cec_driver.c

```diff
@@ -844,11 +845,14 @@ void test_hdmicec_hal_l1_removeLogicalAddress_sinkDevice_negative( void )
     result = HdmiCecRemoveLogicalAddress( handle,  -1 );
     if (HDMI_CEC_IO_INVALID_ARGUMENT  != result) { UT_FAIL("HdmiCecRemoveLogicalAddress failed"); }
 
+    result = HdmiCecAddLogicalAddress( handle, logicalAddress );
+    if (HDMI_CEC_IO_SUCCESS != result) { UT_FAIL("HdmiCecAddLogicalAddress failed"); }
+
     result = HdmiCecRemoveLogicalAddress( handle, logicalAddress );
     if (HDMI_CEC_IO_SUCCESS != result) { UT_FAIL("HdmiCecRemoveLogicalAddress failed"); }
 
     result = HdmiCecRemoveLogicalAddress( handle, logicalAddress );
-    if (HDMI_CEC_IO_ALREADY_REMOVED != result) { UT_FAIL("HdmiCecRemoveLogicalAddress failed"); }
+    if (HDMI_CEC_IO_NOT_ADDED != result) { UT_FAIL("HdmiCecRemoveLogicalAddress failed"); }
```

Comment:
need to raise a separate ticket on this and check the test upgraded based on this problem, you can't commit this.